tests/integration/utils/utils.py

```diff
@@ -13,15 +13,40 @@
 # limitations under the License.
 #
 
+import unittest
+
 import numpy as np
 
 import cunumeric as num
 
 
-def compare_array(a, b):
+class _Dummy(unittest.TestCase):
+    def nop(self):
+        pass
+
+
+_d = _Dummy("nop")
+
+
+def assert_raises(*args, **kwargs):
+    __tracebackhide__ = True  # Hide traceback for py.test
+    return _d.assertRaises(*args, **kwargs)
+
+
+def assert_raises_regex(exception_class, expected_regexp, *args, **kwargs):
+    __tracebackhide__ = True  # Hide traceback for py.test
+    return _d.assertRaisesRegex(
+        exception_class, expected_regexp, *args, **kwargs
+    )
```

Comment:
@robinw0928 whats the specific need here that can't be met using
```
with pytest.raises(...):
    # code that should raise
```

Reply:
Fixed. Thanks.